### PR TITLE
Change cloud.provider from ec2 to aws and from gce to gcp in add_cloud_metadata

### PR DESCRIPTION
**Pull request by kaiyan-sheng**
With the change https://github.com/elastic/ecs/pull/348 in ECS, add_cloud_metadata should change `cloud.provider` from `ec2` to `aws` and from `gce` to `gcp` to match the example in https://www.elastic.co/guide/en/ecs/current/ecs-cloud.html

### CODE REVIEW

File: libbeat/processors/add_cloud_metadata/provider_aws_ec2_test.go [hunk 1]
```diff
@@ -82,7 +82,7 @@ func TestRetrieveAWSMetadata(t *testing.T) {
 
 	expected := common.MapStr{
 		"cloud": common.MapStr{
-			"provider": "ec2",
+			"provider": "aws",
```
Comment (ruflin) [inline libbeat/processors/add_cloud_metadata/provider_aws_ec2_test.go]: I wonder if this is also going to break some of our dashboards?

As I would consider this a bug in 7 I'm good with getting this in even though it's a breaking change.

File: CHANGELOG.next.asciidoc [hunk 2]
```diff
@@ -25,6 +25,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 *Metricbeat*
 
 - Add new option `OpMultiplyBuckets` to scale histogram buckets to avoid decimal points in final events {pull}10994[10994]
+- Change cloud.provider from ec2 to aws in add_cloud_metadata. {issue}10775[10775]
```
Comment (ruflin) [inline CHANGELOG.next.asciidoc]: Could you mention here that this is to align with ECS?

File: libbeat/docs/processors-using.asciidoc [hunk 3]
```diff
@@ -495,7 +495,7 @@ provider and cache the instance metadata.
 
 The following cloud providers are supported:
 
-- Amazon Elastic Compute Cloud (EC2)
+- Amazon Web Services (AWS)
```
Comment (ruflin) [inline libbeat/docs/processors-using.asciidoc]: Interesting that even in the docs we mentioned EC2.

**Comment by kaiyan-sheng** [comment]
@ruflin @exekias  Thanks for the review. Quick question, similar to this https://www.elastic.co/guide/en/ecs/current/ecs-cloud.html, GCE probably should be GCP instead? hmm

File: CHANGELOG.next.asciidoc (hunk 2 above)
Comment (kaiyan-sheng) [reply CHANGELOG.next.asciidoc]: done, thanks!

**Comment by ruflin** [comment]
jenkins, test this

**Comment by exekias** [comment]
+1 to changing GCE to GCP too, hopefully in the same version

**Comment by ruflin** [comment]
@kaiyan-sheng Could you double check that this does not break any dashboards / visualisatins?

**Comment by kaiyan-sheng** [comment]
> @kaiyan-sheng Could you double check that this does not break any dashboards / visualisatins?

@ruflin Sorry I forgot to respond to you on this earlier. I did a search in metricbeat dashboards and filebeat dashboards to see if there's anywhere needed cloud.provider=ec2/gce and I didn't see any. So I assume it's safe here. And to be more sure about this, I'm trying to setup metricbeat/kibana/es on GCP to actually test it.

**Comment by kaiyan-sheng** [comment]
> LGTM. This will also need backports I assume?

Yes! Sorry I forgot to add the labels.

File: CHANGELOG.next.asciidoc [hunk 4]
```diff
@@ -27,6 +27,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 *Metricbeat*
 
 - Add new option `OpMultiplyBuckets` to scale histogram buckets to avoid decimal points in final events {pull}10994[10994]
+- Change cloud.provider from ec2 to aws and from gce to gcp in add_cloud_metadata to align with ECS. {issue}10775[10775] {pull}11687[11687]
```
Comment (andrewkroh) [inline CHANGELOG.next.asciidoc]: This is a breaking change affect all Beats.

**Comment by kaiyan-sheng** [comment]
@andrewkroh Yeah my bad... I didn't quite realize the timing there...

**Pull request merged**